packages/Alert/src/alert.scss

```diff
@@ -14,34 +17,86 @@
   --ids-alert-box-shadow: #{tokens.$shadow-6};
   --ids-alert-background-color-light: #{tokens.$samoyed};
   --ids-alert-background-color-darker: #{tokens.$grey-100};
-  --ids-alert-border-left-width: 0.6rem;
+  --ids-alert-border-default-width: 0 0 0 0.6rem;
+  --ids-alert-border-width: 0 0 0 0.6rem;
+  --ids-alert-content-margin-top: #{tokens.$space-1};
   --ids-alert-text-color-title: #{tokens.$grey-800};
   --ids-alert-text-color-body: #{tokens.$grey-600};
   --ids-alert-icon-size-small: 2.6rem;
   --ids-alert-icon-size-medium: 4.0rem;
-  --ids-alert-link-color: #{tokens.$electric-blue-500};
   --ids-alert-btn-margin-top: #{tokens.$space-3};
   --ids-alert-metadata-color: #{tokens.$grey-600};
 
   /* Announcement */
-  --ids-alert-color-announcement: #{tokens.$seaweed-500};
+  --ids-alert-border-color-announcement: #{tokens.$seaweed-500};
```

Comment:
Can't we map to semantic colors?

Reply:
That's the original Igloo style. There are no semantic tokens.